easybuild/easyconfigs/p/PyCUDA/PyCUDA-2020.1-fosscuda-2020b.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonBundle'
+
+name = 'PyCUDA'
+version = '2020.1'
+
+homepage = 'https://mathema.tician.de/software/pycuda'
+description = "PyCUDA lets you access Nvidia’s CUDA parallel computation API from Python."
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('Boost.Python', '1.74.0'),
+    ('Mako', '1.1.3'),
+]
+
+local_preinstallopts = "./configure.py --cuda-root=$EBROOTCUDA --boost-inc-dir=$EBROOTBOOST/include/boost "
+local_preinstallopts += "--boost-lib-dir=$EBROOTBOOST/lib --no-use-shipped-boost "
+local_preinstallopts += "--boost-python-libname=boost_python38 && "
+
+sanity_pip_check = True
+use_pip = True
+
+exts_default_options = {
+    'source_tmpl': '%(namelower)s-%(version)s.tar.gz',
+    'source_urls': ['https://pypi.python.org/packages/source/%(nameletterlower)s/%(namelower)s'],
```

Comment:
Any reason you're not using SOURCELOWER_TAR_GZ and PYPI_LOWER_SOURCE here?

Reply:
I'm wondering why that isn't the default for python packages already...

Reply:
because it is just as common to  have PYPI_SOURCE or SOURCE_TAR_GZ ?